bin/templates/cordova/lib/check_reqs.js

```diff
@@ -355,16 +355,16 @@ module.exports.check_android_target = function (originalError) {
 // Returns a promise.
 module.exports.run = function () {
     return Q.all([this.check_java(), this.check_android()]).then(function (values) {
-        console.log('ANDROID_HOME=' + process.env['ANDROID_HOME']);
-        console.log('JAVA_HOME=' + process.env['JAVA_HOME']);
```

Comment:
Ditto: I think this is information that can help a user in case something goes wrong.

Reply:
Here I don't really understand the code of `check_java` - is `JAVA_HOME` really used? Or is this misleading?

Reply:
JAVA_HOME is used:

<https://github.com/apache/cordova-android/blob/d9c08f12a7bceb47ee97ca8ca0e75548e78f999d/bin/templates/cordova/lib/check_reqs.js#L147>

<https://github.com/apache/cordova-android/blob/d9c08f12a7bceb47ee97ca8ca0e75548e78f999d/bin/templates/cordova/lib/check_reqs.js#L152>

and some other places.

Reply:
Another thing is that `check_reqs.js` will actually set `process.env['JAVA_HOME']` if necessary to keep track of the actual Java JDK location. I think we should fix `check_reqs.js` to track this information in an internal variable instead. I would favor working on this in a separate PR, with some more leisure.

Reply:
I think this is misleading: Isn't it actually the output of `javac -version` that defines the version that is used for comparison? What exactly does the code _do_ with `JAVA_HOME` there? Is this just for the final comparison at https://github.com/apache/cordova-android/blob/d9c08f12a7bceb47ee97ca8ca0e75548e78f999d/bin/templates/cordova/lib/check_reqs.js#L209-L212? Why is `JAVA_HOME` relevant?

Reply:
JAVA_HOME can help determine where javac is run from. I would consider that to be relevant.

Reply:
From <https://github.com/apache/cordova-android/issues/623#issuecomment-455327547> my understanding is that the logging of JAVA_HOME may be misleading. So I would agree to take it out, at least for now. I would like to add some more detailed logging, in a non-misleading manner, someday in the future. I will mark this conversation as resolved.